### ISSUE TITLE
Prepare Chrome Expression Whitespacing for IntegerExpression (& Readability)

### ISSUE THREAD

**Issue by atlimit8** [OpenRA/OpenRA#13153]
Replacing `Evaluator` with `IntegerExpression` requires reformatting chrome expressions to have the required whitespace around binary operators. Closing this does not require actually replacing `Evaluator` with `IntegerExpression`, just reformatting the MiniYaml.

Examples:
 * `(PARENT_RIGHT - WIDTH)/2` => `(PARENT_RIGHT - WIDTH) / 2`
 * `PARENT_RIGHT-35` => `PARENT_RIGHT - 35`
 * `PARENT_BOTTOM-45-50` => `PARENT_BOTTOM - 45 - 50`
 * `PARENT_RIGHT-15-WIDTH` => `PARENT_RIGHT - 15 - WIDTH`

This is simply a matter of finding the expressions and using the spacebar to add the needed spaces.

**Comment by daVoodooShuffle** [comment]
Could I do this one. This my first time contributing.

**Comment by atlimit8** [comment]
@daVoodooShuffle, sure.

**Comment by atlimit8** [comment]
Look in the chrome directories (folders) of each mod (cnc, common, d2k, modchooser, ra, & rs).

**Comment by daVoodooShuffle** [comment]
No problem. I'll get that done as soon as I can. 
Thanks

**Comment by daVoodooShuffle** [comment]
I've made the changes to all the appropriate YAML files. However, I'm not to sure what you mean by reformatting the MiniYaml class. What needs to be changed there to ensure this all works correctly?

**Comment by atlimit8** [comment]
MiniYaml is just the name of our custom YAML like language in our .yaml files. At a glance, it appears that your commit covers the issue. When I said reformatting the MiniYaml, that spacing was what I was talking about; none of the C# code needs to be changed to closes this issue.

It would be nice if you changed the top line of the commit message to "Closes #13153", which can be done by typing `git commit --amend` in the command line. If you use a graphical tool, you will have to look at the documentation. In the future, you might want to create branches for your pull requests. It allows for working on multiple sets of changes at once. As long as no conflicting changes are made to bleed on https://github.com/OpenRA/OpenRA, you should be fine for now.

**Comment by daVoodooShuffle** [comment]
Alright thanks for the heads up. I'm still getting my head around git. I figured I did something wrong there. Is there anything else I need to do to close this issue?

**Comment by atlimit8** [comment]
Actually, the three commits need to be reduced to one first, then you can open a pull request.
You can see your three commits [here](https://github.com/daVoodooShuffle/OpenRA/commits/bleed).

Here is some surgery for your repo:

This is just creating a diff (comparison) file to delete if everything works:
```
git diff 60bc114e391dea7a3e0525b73182b6ece5921de6...bleed>../OpenRA.tmp.diff
```
These lines create a backup branch to delete if everything works:
```
git checkout bleed
git checkout -b backup
```
These lines reset bleed back to before your changes:
```
git checkout bleed
git reset --hard 60bc114e391dea7a3e0525b73182b6ece5921de6
git push origin +bleed
```
This line creates a new branch for the changes:
```
git checkout -b fix-13153
```
This appends your "Closes #13153 Issue" commit to the `fix-13153` branch:
```
git cherry-pick 95e1e002687cd01cecb69a9556d1f6c35e00256b
```
You can check for your changes with:
```
 git diff 95e1e002687cd01cecb69a9556d1f6c35e00256b
```
The result should be nothing.

Please shorten "Closes #13153 Issue" to "Closes #13153" with:
```
git commit --amend
```
This line publishes the new branch to your GitHub fork:
```
git push origin fix-13153
```
Go to https://github.com/daVoodooShuffle/OpenRA/tree/fix-13153
Click on "New pull request".
Use this as the title: "Reformatted Chrome Expression Whitespacing".

This line removes the backup branch
```
git branch -D backup
```
Lastly, delete OpenRA.tmp.diff in the parent directory.

---
Do the following for updating:
```
git checkout bleed
git pull upstream bleed
```
Then to update your branch:
```
git checkout fix-13153
git rebase bleed
git push origin +fix-13153
```
